plugins/sqlfluff-plugin-example/setup.py

```diff
@@ -0,0 +1,15 @@
+"""Setup file for example plugin."""
```

Comment:
I wonder if it may be easier for folks to develop plugins if we provided the example plugin as a separate "GitHub Template Repository" that they could fork, maybe with some templating to replace the word "example", etc.

https://github.blog/2019-06-06-generate-new-repositories-with-repository-templates/
https://rclayton.silvrback.com/templatizing-github-template-repos

Reply:
That would be really useful, maybe it could be a separate issue/PR? I think it wouldn't be that hard today to just copy+paste the example plugin directory

Reply:
https://github.com/sqlfluff/sqlfluff/issues/794

Reply:
Sure, a separate PR is fine.